templater/jobs/presubmit/eks-distro/cni-1-X-presubmits.yaml

```diff
@@ -1,7 +1,10 @@
-jobName: cni-plugins-presubmit
-runIfChanged: ^build/lib/.*|Common.mk|projects/containernetworking/plugins/.*
+jobName: cni-plugins-{{ .releaseBranch }}-presubmit
+runIfChanged: ^build/lib/.*|Common.mk|projects/containernetworking/plugins/(build|Makefile|{{ .releaseBranch }})
```

Comment:
The other templates include `EKS_DISTRO_BASE_TAG_FILE|` at the start. Do we need this too for CNI?

Reply:
No we don't, since this project doesn't have any container image artifacts (and it's the only one that doesn't).

Reply:
cool thanks!